lib/ui/page/home/page/chat/view.dart

```diff
@@ -829,13 +819,12 @@ class ChatView extends StatelessWidget {
                 context,
                 c,
                 item: element,
-                overlay:
-                    e.value.author.id != c.me || element is ChatInfoElement,
                 child: ChatItemWidget(
                   chat: c.chat!.chat,
                   item: e,
                   me: c.me!,
                   avatar: !previousSame,
+                  appendAvatarPadding: !c.selecting.value,
```

Comment:
I've also noticed that text gets squashed when selection is enabled. This causes all messages to rebuild. Is there anything that can be done to prevent such behaviour? Doesn't it hits your eye too? For me as a user this look awful and buggy as hell. I would suggest that this product is raw and still is beta when looking at such message squashing.


https://github.com/user-attachments/assets/8ac8965c-8a7f-4085-8b9f-a3cff04fd4b1

Why do the messages squash? Isn't here enough space for `SelectionDot`s to fit in the space left at the left and right edges of the screen?

<img width="591" height="541" alt="Screenshot 2025-09-17 at 12 32 16" src="https://github.com/user-attachments/assets/580afd17-a4e8-4dfc-9b3d-3811cfdeebfa" />